orchagent/fdborch.cpp

```diff
@@ -1601,7 +1612,7 @@ bool FdbOrch::removeFdbEntry(const FdbEntry& entry, FdbOrigin origin)
     fdb_entry.bv_id = entry.bv_id;
 
     status = sai_fdb_api->remove_fdb_entry(&fdb_entry);
-    if (status != SAI_STATUS_SUCCESS)
+    if ((status != SAI_STATUS_SUCCESS) && (status != SAI_STATUS_ITEM_NOT_FOUND))
```

Comment:
Can you also add a comment here as why this could be a case that we should allow to move on and not treated as an error case.

Reply:
Thanks @gechiang . I have added comments now.